OpenIGTLinkIF/MRML/vtkMRMLIGTLConnectorNode.h

```diff
@@ -50,16 +50,17 @@ class VTK_SLICER_OPENIGTLINKIF_MODULE_MRML_EXPORT vtkMRMLIGTLConnectorNode : pub
   enum
   {
     ConnectedEvent = 118944,
-    DisconnectedEvent = 118945,
-    ActivatedEvent = 118946,
-    DeactivatedEvent = 118947,
-    ReceiveEvent = 118948, // deprecated. it was for query response, OpenIGTLinkIO doesn't support query event. it is replaced with command message
-    NewDeviceEvent = 118949,
-    DeviceModifiedEvent = 118950,
-    RemovedDeviceEvent= 118951,
-    CommandReceivedEvent = 119001, // Command received
-    CommandResponseReceivedEvent = 119002, // Command response
-    CommandCompletedEvent = 119003, // Command completed (could be response received, or expired, or cancelled)
+    DisconnectedEvent,
+    ActivatedEvent,
+    DeactivatedEvent,
+    ReceiveEvent, // deprecated. it was for query response, OpenIGTLinkIO doesn't support query event. it is replaced with command message
```

Comment:
Is this comment still true?

Reply:
I'm not sure. I don't know what the original use case was.